config/nginx-vanilla.conf

```diff
@@ -1,11 +1,5 @@
 # example config for server's public facing nginx
 
-# SSL config
-ssl_certificate /etc/letsencrypt/live/TLD_NAME/fullchain.pem;
-ssl_certificate_key /etc/letsencrypt/live/TLD_NAME/privkey.pem;
-#include /etc/letsencrypt/options-ssl-nginx.conf;
-#ssl_dhparam /etc/letsencrypt/ssl-dhparams.pem;
-
 # Redirect to https
 server {
```

Comment:
The reason why I removed it is because certbot automatically adds it when you deploy for the first time. However, if you have pre-existing SSL directives and no key files, certbot will fail.

Reply:
gotcha... yes, that makes sense. good idea